termwiz/src/caps/mod.rs

```diff
@@ -302,6 +314,12 @@ impl Capabilities {
         self.hyperlinks
     }
 
+    /// Does the terminal support hyperfiles?
+    /// See <https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda>
```

Comment:
this spec doesn't apply to hyperfiles

Reply:
Right, I thought I made up the term hyperfile given the hyperlink name. I could delete this line

Reply:
done